crypto/bio/bss_mem.c

```diff
@@ -228,28 +265,32 @@ static long mem_ctrl(BIO *b, int cmd, long num, void *ptr)
     long ret = 1;
     char **pptr;
 
```

Comment:
remove this blank line; all local vars in one blank, with a blank line *after* them.